tests/function-controller/testsuite/testsuite.go

```diff
@@ -189,6 +258,18 @@ func (t *TestSuite) Cleanup() {
 	err = t.broker.Delete()
 	failOnError(t.g, err)
 
+	err = t.servicebindingusage.Delete()
+	failOnError(t.g, err)
+
+	err = t.servicebinding.Delete()
```

Comment:
ServiceBindingUsage has as ownerReference ServiceBinding, so you can delete only SB, but this is not a problem :)

Reply:
I create that manually, so I there's any owner reference

Reply:
ok, I just learned that sbu controller does it for us, nice

still, I think we could leave it here, just to be explicit